docs/Getting-Started.md

```diff
@@ -536,4 +536,55 @@ Use the `ViewportTransform` dependency property to have the grid move with the v
 </Window>
 ```
 
-> Tip: Right-click and drag the screen around to move the view and use the mouse wheel to zoom in and out.
\ No newline at end of file
+> Tip: Right-click and drag the screen around to move the view and use the mouse wheel to zoom in and out.
```

Comment:
The wiki contains documentation only about Nodify because that's the published package on NuGet. The Nodify.Shared project was developed as a utility project for quickly developing example apps and was not meant to be used in production. However, the best place for this type of documentation would be a new README file in the Examples folder.